include/Editor.h

```diff
@@ -80,7 +80,7 @@ private slots:
 	///
 	/// \param	record	If set true, the editor's toolbar will contain record
 	///					buttons in addition to the play and stop buttons.
-	Editor(bool record = false, bool record_step = false);
+	Editor(bool record = false, bool recordAccompany = false, bool record_step = false);
```

Comment:
If you think it makes sense, change `record_step` to `recordStep` and update the doxygen comment above to reflect all 3 parameters.